build.gradle

```diff
@@ -118,3 +124,28 @@ subprojects {
         }
     }
 }
+
+task jacocoAggregateReport(type: org.gradle.testing.jacoco.tasks.JacocoReport) {
```

Comment:
:notebook: I tried moving this to its own file. The use of the `subprojects` object prevents it from being used outside of the same file as where `subprojects` are declared.